example/config/README.md

```diff
@@ -0,0 +1,249 @@
+
+
+### Feature Generation
+The aggregate features to generate for each train/test split. Implemented by wrapping [collate](https://github.com/dssg/triage/tree/config_doc/src/triage/component/collate). Most terminology here is taken directly from [collate](https://github.com/dssg/triage/tree/config_doc/src/triage/component/collate).
+
+Each entry describes a collate.SpacetimeAggregation object, and the arguments needed to create it. Generally, each of these entries controls the features from one source table, though in the case of multiple groups may result in multiple output tables.
+
+Rules specifying how to handle imputation of null values must be explicitly defined in your config file. These can be specified in two places: either within each feature or overall for each type of feature (aggregates_imputation, categoricals_imputation, array_categoricals_imputation). In either case, a rule must be given for each aggregation function (e.g., sum, max, avg, etc) used, or a catch-all can be specified with `all`. Aggregation function-specific rules will take precedence over the `all` rule and feature-specific rules will take precedence over the higher-level rules. Several examples are provided below.
+
+Available Imputation Rules: 
+- `mean`: The average value of the feature (for SpacetimeAggregation the mean is taken within-date).
+- `constant`: Fill with a constant value from a required `value` parameter.
+- `zero`: Fill with zero.
+- `null_category`: Only available for categorical features. Just flag null values with the null category column.
+- `binary_mode`: Only available for aggregate column types. Takes the modal value for a binary feature.
+- `error`: Raise an exception if any null values are encountered for this feature.
+
+    
+- `feature_aggregations`: 
+    - `prefix`: prefix given to the resultant tables
+    - `from_obj`: from_obj is usually a source table but can be an expression, such as a join (ie ```"cool_stuff join other_stuff using (stuff_id)"```)
+    - `knowledge_date_column`: The date column to use for specifying which records to include in temporal features. It is important that the column used specifies the date at which the event is known about, which may be different from the date the event happened.
+    - `aggregates_imputation`: top-level imputation rules that will apply to all aggregates functions can also specify `categoricals_imputation` or `array_categoricals_imputation`. You must specified at least one of the top-level or feature-level imputation to cover ever feature being defined.
+        - `all`: The `all` rule will apply to all aggregation functions, unless overridden by more specific one
+            - `type`: every imputation rule must have a `type` parameter, while some (like 'constant') have other required parameters (`value` here)
+            - `value`
+        - `max`: specifying `max` here will take precedence over the `all` rule for aggregations using a MAX() function
```

Comment:
Perhaps list out the functions... or make this more generic, like any function will override `all` for all aggregations using that function